experimental/generation/generator/packages/library/test/forms/unittest_array_personName.schema

```diff
@@ -0,0 +1,21 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "type": "object",
+    "properties": {
+        "NameArray": {
+            "type": "array",
+            "items": {
+                "type": "string"
+            },
+            "$entities": [
+                "personName"
+            ]
```

Comment:
Do $ref to personName.schema inside items.

Reply:
Do you mean the NameArray property could be written as follows:    
    "NameArray": {
            "type": "array",
            "items": {
                "$ref" : "template:personname.schema"
            }
        }